src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -703,6 +704,45 @@ void AppHost::_initialResizeAndRepositionWindow(const HWND hwnd, til::rect propo
     LOG_LAST_ERROR_IF(!succeeded);
 }
 
+// Method Description:
+// - Resize the window when window size changed signal is received.
+// Arguments:
+// - hwnd: The HWND of the window we're about to resize.
+// - newSize: The new size of the window in pixels.
+// Return Value:
+// - None
+void AppHost::_resizeWindow(const HWND hwnd, til::size newSize)
+{
+    til::rect windowRect{ _window->GetWindowRect() };
+    UINT dpix = _window->GetCurrentDpi();
+
+    const auto islandWidth = Utils::ClampToShortMax(lrintf(static_cast<float>(newSize.width)), 1);
+    const auto islandHeight = Utils::ClampToShortMax(lrintf(static_cast<float>(newSize.height)), 1);

```

Comment:
The float-cast + `lrintf` seems redundant. I think this could be just
```cpp
const auto islandWidth = Utils::ClampToShortMax(newSize.width, 1);
const auto islandHeight = Utils::ClampToShortMax(newSize.height, 1);
```